apps/web/src/components/UserProfile/UserProfile.tsx

```diff
@@ -130,427 +43,168 @@ const UserProfile = ({
-              <TableOfContents.ListItem>
-                <TableOfContents.AnchorLink
-                  id={PAGE_SECTION_ID.ACCOUNT_AND_PRIVACY}
-                >
-                  {intl.formatMessage({
-                    defaultMessage: "Account and privacy settings",
-                    id: "",
-                    description:
-                      "Title of the Account and privacy settings link section",
-                  })}
-                </TableOfContents.AnchorLink>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.ABOUT}>
+                {intl.formatMessage(navigationMessages.aboutMe)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.DEI}>
+                {intl.formatMessage(
+                  navigationMessages.diversityEquityInclusion,
+                )}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.LANGUAGE}>
+                {intl.formatMessage(navigationMessages.languageInformation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.GOVERNMENT}>
+                {intl.formatMessage(navigationMessages.governmentInformation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_LOCATION}>
+                {intl.formatMessage(navigationMessages.workLocation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_PREFERENCES}>
+                {intl.formatMessage(navigationMessages.workPreferences)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink
+                id={PAGE_SECTION_ID.CAREER_TIMELINE_AND_RECRUITMENT}
+              >
+                {intl.formatMessage(
+                  navigationMessages.careerTimelineAndRecruitment,
+                )}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.SKILL_SHOWCASE}>
+                {intl.formatMessage(navigationMessages.skillShowcase)}
+              </TableOfContents.AnchorLink>
             </TableOfContents.ListItem>
-          )} */}
           </TableOfContents.List>
         </TableOfContents.Navigation>
```

Comment:
This is a larger change but, I am wondering if we want to move the print button here so we don't have all that extra  whitespace to accommodate it? :thinking: 

Not necessary so feel free to ignore.

![screenshot_24-Jun-2024_14-18-1719253125](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/84e5afe6-a7cb-4681-b641-2461d26acdc1)